examples/layer_shell.rs

```diff
@@ -0,0 +1,111 @@
+#![allow(warnings)]
+
+use bevy::{color::palettes::basic::*, prelude::*, winit::WinitPlugin};
+use bevy_smithay::{
+    SmithayPlugin,
+    prelude::{Anchor, Layer, LayerShellSettings},
+};
+
+fn main() {
+    App::new()
+        .add_plugins((
+            DefaultPlugins.build().disable::<WinitPlugin>(),
+            SmithayPlugin,
+        ))
+        // Only run the app when there is user input. This will significantly reduce CPU/GPU use.
+        .insert_resource(LayerShellSettings {
```

Comment:
add inline comments for each of these layershell properties considering this is an example